spring-boot-project/spring-boot-actuator-autoconfigure/src/test/java/org/springframework/boot/actuate/autoconfigure/metrics/test/MetricsRun.java

```diff
@@ -76,7 +77,8 @@
 			RabbitMetricsAutoConfiguration.class, CacheMetricsAutoConfiguration.class,
 			DataSourcePoolMetricsAutoConfiguration.class,
 			RestTemplateMetricsAutoConfiguration.class,
-			WebFluxMetricsAutoConfiguration.class, WebMvcMetricsAutoConfiguration.class);
+			WebFluxMetricsAutoConfiguration.class, WebMvcMetricsAutoConfiguration.class,
+			JerseyServerMetricsAutoConfiguration.class);
```

Comment:
This is just for completeness sake in case somebody is going to use the `MetricsRun` tooling in other tests. It's not used in the Jersey auto-configuration tests.